### DISCUSSION TITLE
new subscriber - Evernote "conversion" - trying to set up Folders (Notebooks) vs tags in Standard Notes

### DISCUSSION THREAD

**Discussion by BradfordMoi** [standardnotes/forum#2938] [General]
Been using Evernote for 10+ years, found Standard Notes - looking to give it a go.  subscribed to professional plan today to get full functionality.  

have an extensive notebook / Tag structure set up in Evernote.  Rely on tags for most comprehensive organization, but use notebooks as well - 
* Notebooks 
** Inbox
** FilingCabinet 
** versions of both for personal and biz 

* use nested tags then to further classify - 
** financial 
*** biz expense
*** receipts 
*** tax items 
etc 

need to find info / video on how to emulate that set up in Standard Notes - cannot find how to add Folders vs Tags?  only looks like I am adding tags.  

* how do I create folders (notebooks) - 
* Inbox
* FilingCabinet 
etc 

then use the tags as well?  

thank you for any help and support.  

Using - 
* Web Version 
** Chrome
* Mac OSx app
** Macbook Pro and Air 
* iOSx app 
** iPhone 11
* Android app
** Samsung Fold 4

**Comment by shompoe** [comment]
Standard Notes does not have folders or notebooks. Though if I'm not mistaken there was talk of implementing notebooks in the future.

What other people do in your situation is to tag everything either personal or business etc. and add whatever other tags as well. Then you can create customized views to display notes which have certain tags whilst filtering out either personal or business.  Or another approach is to create personal and business etc. high level tags and then any other tags as sub-tags under those. 

Alternatively of course you could create separate accounts, both of which can be linked to your subscription.

Not the answer you wanted but hope it helps.

**Comment by BradfordMoi** [comment]
@shompoe 

appreciate the prompt response.  got it.  understand.  will hack my way through it for now.  thanks again.

**Comment by fra-gadio** [comment]
Did you manage to find a solution? I'm trying to do the same

**Comment by myreli** [comment]
Hello @BradfordMoi, welcome to Standard Notes! 😄  

Folders on SN work exactly like tags, they are in fact nested tags. My setup, for example, looks kind of like this: 

<img width="499" alt="image" src="https://user-images.githubusercontent.com/17554234/221377539-f6488850-218b-4448-a301-8994b52b9a6a.png">

You can also drag-and-drop tags to create the folder structure. 

If you would like to further separate your work tags, SN doesn't currently offer a "notebook" feature, but you can create a separate account and share your subscription with yourself. I have my main account and this demo account
 
<img width="649" alt="image" src="https://user-images.githubusercontent.com/17554234/221377706-f550fa37-3e69-46c6-8133-c1e605dc7c1a.png">
 
Be sure to import your [Evernote](https://standardnotes.com/help/10/how-can-i-import-my-notes-from-evernote) content to make the best out of it. 

@fra-gadio let me know if this helps you as well. 🙂

**Comment by fra-gadio** [comment]
@myreli thank you for your reply. 

But let's say in your example you wanted to add 2 sub tags called #2022 and #2023 to the Documents tag to further categorize your documents.  Those would have the same name as the other tags you already have.  
Isn't the whole point of tags that they're unique and not redundant?

**Comment by myreli** [comment]
Yes, that's a good perspective @fra-gadio. I guess that is the downside of having Folders and Tags as the same entity. 

In that case, I would organize my notes using Tags and Views, but let go of Folders. Or duplicate the tags. 

**Duplicated Tags**: I would create top-level tags for everything that I want to group uniquely `#2023` and `#2022` and then duplicate this in any folder that I need. The downside is having to type the tags twice: one for `#Journaling.2023` and the other for `#2023`
<img width="218" alt="image" src="https://user-images.githubusercontent.com/17554234/221394036-4183b9cb-ee5b-4696-ab10-3bf7fea4e4b3.png">
 

**Tags + Views**: I would tag things `#Journaling` `#2023` and `#Documents` `#2023`. And then create multiple views for that.

<img width="218" alt="image" src="https://user-images.githubusercontent.com/17554234/221393907-5d8bfd9f-5172-41aa-b4d9-40f8b4968330.png">
 
 But I can see that wouldn't be feasible for all workflows, but there are a lot of creative ways around this. Looking forward to see how you organize it 🙂
